dynawo/sources/Solvers/AlgebraicSolvers/DYNSolverKINAlgRestoration.cpp

```diff
@@ -325,17 +303,9 @@ SolverKINAlgRestoration::solve(bool noInitSetup) {
 
   // yScale
   yScale_.assign(indexY_.size(), 1.0);
-  if (mode_ == KIN_NORMAL) {  // variables = YAlg
-    for (unsigned int i = 0; i < indexY_.size(); ++i) {
-      if (std::abs(y0_[indexY_[i]]) > RCONST(1.0)) {
-        yScale_[i] = 1. / std::abs(y0_[indexY_[i]]);
-      }
-    }
-  } else if (mode_ == KIN_YPRIM) {  // variables = YP
-    for (unsigned int i = 0; i < indexY_.size(); ++i) {
-      if (std::abs(yp0_[indexY_[i]]) > RCONST(1.0)) {
-        yScale_[i] = 1. / std::abs(yp0_[indexY_[i]]);
-      }
+  for (unsigned int i = 0; i < indexY_.size(); ++i) {
+    if (std::abs(vYy_[indexY_[i]]) > RCONST(1.0)) {
+      yScale_[i] = 1. / std::abs(vYy_[indexY_[i]]);
```

Comment:
Why has the denominator changed?

Reply:
Here, when we are with KIN_NORMAL vYy is y0 and with KIN_YPRIM vYy is yp0 so the denominator finally didn't change.